### DISCUSSION TITLE
Slideshow sharing link

### DISCUSSION THREAD

**Discussion by SelfSufficient08** [photoprism/photoprism#4112] [General]
Is there a way to add flags to a URL that would auto open the shared moments or album in a slideshow that plays immediately?  And if that is possible, is there a way to adjust how long it stays on each photo?  I'd like to use this type of feature to feed digital photo frames (fire tablets).
I use picapport for sharing slideshows.  It works great but it lacks the facial recognition and moments aspect.  
An example from picapport url is: http://192.168.1.100:8080/picapport#slideshow?query=florida+jordan&sort=randomsmart&viewtime=40&autostart=true
Hoping I can replicate something like that in this.  Picapport is great but it relies on the naming of files and folders which is inefficient long term. 
thanks !

**Comment by graciousgrey** [comment]
That's not possible yet but something we would love to support in the future. 

Related issues:
- https://github.com/photoprism/photoprism/issues/2016
- https://github.com/photoprism/photoprism/issues/1307
